nullaway/src/main/java/com/uber/nullaway/handlers/RestrictiveAnnotationHandler.java

```diff
@@ -78,6 +78,28 @@ public boolean onOverrideMayBeNullExpr(
     return exprMayBeNull;
   }
 
+  @Override
+  public ImmutableSet<Integer> onUnannotatedInvocationGetExplicitlyNullablePositions(
+      NullAway analysis,
+      VisitorState state,
+      Symbol.MethodSymbol methodSymbol,
+      ImmutableSet<Integer> explicitlyNullablePositions) {
```

Comment:
Yes. Also is it possible that in future we might want to handle nullness of return values in LibraryModelsHandler?

Reply:
You mean for overriding? The case of handling _use_ of null return values from library models is done by `onOverrideMayBeNullExpr`. 

It might be worth adding `onUnannotatedInvocationGetExplicitlyNonNullReturn` to that handler as well, though, want to do it as another short PR?

Reply:
Yeah, I'll go for another short PR